package.json

```diff
@@ -18,10 +18,13 @@
     "@angular/compiler": "~13.3.0",
     "@angular/core": "~13.3.0",
     "@angular/forms": "~13.3.0",
+    "@angular/localize": "~13.3.0",
```

Comment:
this is used for translations. Im not sure If this will have more than one language